_data/projects/dockerfile-archieve.yml

```diff
@@ -0,0 +1,16 @@
+
+
+---
+name: Dockerfile Archive
+desc: A repository of all Dockerfiles in one place to make it easier to find.
+site: https://github.com/darkmatter18/dockerfile-archive
+tags:
+- docker
+- dockerfile
+- container
+- python
+- javascript
+- cloud
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/darkmatter18/dockerfile-archive/labels/up%20for%20grabs
```

Comment:
There are no listed issues currently with this label, which may mean this won't be listed on the site. Do you plan to add some tasks @darkmatter18 ?

Reply:
@shiftkey I'll add some tass today

Reply:
@shiftkey I added some tasks. Will add some more